main/src/ca/uwaterloo/flix/language/ast/ResolvedAst.scala

```diff
@@ -364,7 +364,7 @@ object ResolvedAst {
     case class Implicit(name: Name.Ident, sym: Symbol.UnkindedTypeVarSym, loc: SourceLocation) extends TypeParam
   }
 
-  case class TraitConstraint(head: Ast.TraitConstraint.Head, tpe: UnkindedType, loc: SourceLocation)
+  case class TraitConstraint(head: shared.TraitConstraint.Head, tpe: UnkindedType, loc: SourceLocation)
```

Comment:
Just `TraitConstraint.Head`

Reply:
It clashes with `ResolvedAst.TraitConstraint`